exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -45,6 +45,8 @@ drill.client: {
   supports-complex-types: true
 }
 
+drill.home: "/tmp"
```

Comment:
Changed to drill.home to drill.dfs-home.
The problem why I set value to /tmp is since I am not quite sure to which value to set it.
If I set for example, /user/drill drillbit will fail at start up since currently we don't have such directory and user that runs drillbit usually doesn't have rights to create directory from /.